contracts/.env.example

```
@@ -1,4 +1,4 @@
 INFURA_ID=zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz
 MNEMONIC=here is where your twelve words mnemonic should be put my friend
 DEPLOY_GSN=false
-ETHERSCAN_API_KEY="YOUR_API_KEY"
\ No newline at end of file
+ETHERSCAN_VALIDATION_API_KEY="YOUR_API_KEY"
```

just to be more precise on the use case for the API key
```suggestion
ETHERSCAN_VERIFICATION_API_KEY="YOUR_API_KEY"
```